tools/mason/SpecParser.chpl

```diff
@@ -139,7 +139,10 @@ private proc parseSpec(ref tokenList: list(string)): 4*string throws {
     if package == '' {
       const pkgSplit = toke.split('@');
       package = pkgSplit[0];
-      packageVersion = pkgSplit[1];
+      if pkgSplit.size > 1 then packageVersion = pkgSplit[1];
+      else throw new owned MasonError("Mason resolved to <pkg@version%clang>"+
```

Comment:
Oh ! I misunderstood the error message described in issue because of the quotes. Sorry for delays on my side. Too many university assignments. However, I'll fix all my unmerged PRs by today !

Reply:
No worries, and no rush! Your contributions are appreciated.